src/main/java/uk/gov/hmcts/reform/sscs/model/hearings/CaseCategory.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.sscs.model.hearings;
```

Comment:
Can you change the package to hearing not hearings, as the endpoint is hearing and there is another for hearings

Reply:
Maybe single.hearing would be clearer

Reply:
okay Will make it as single.hearing